src/task3/task3_1.js

```diff
@@ -0,0 +1,16 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+
+import userRouter from './routers/userRouter';
+import { SERVER_PORT } from './config/global';
+import errorHandlerMiddleWare from './middlewares/errorHandlerMiddleWare';
+
+const app = express();
+
+app.use(bodyParser.urlencoded());
+
+app.get('/', (req, res) => res.send('Server is running...'));
+app.use('/users', userRouter);
```

Comment:
Consider making the above app.get use the same approach you're utilising here - a separate router (default router with get handler) - to avoid inconsistency among your route handling logic.